odoo/cli/neutralize.py

```diff
@@ -35,6 +35,7 @@ def run(self, args):
                 env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
                 for model in env.values():
                     model._neutralize()
+                env['ir.config_parameter'].set_param('is_neutralized', True)
```

Comment:
I haven't tested it properly, but I think we should add an explicit registry signaling after setting the param. It's normally done automatically at the end of the request cycle, but there is no request here.
See also the comment on the related PR: https://github.com/odoo/odoo/pull/85192#discussion_r812897139

```suggestion
                env['ir.config_parameter'].set_param('is_neutralized', True)
                env.registry.signal_changes()
```

Reply:
I've tested it together with the other PR applied and can confirm that this works.